docs/README.md

```diff
@@ -132,7 +132,7 @@ Docker images are only availabe on Linux.
 3. Clone `avatarify` and install its dependencies (v4l2loopback kernel module):
 ```bash
 git clone https://github.com/alievk/avatarify-python.git
-cd avatarify
+cd avatarify-python
 bash scripts/install_docker.sh
 ```
 4. Build the Dockerfile:
```

Comment:
There's a "cd avatarify" on line 140 in docker instructions, but it seems like I can't comment lines that are too far from your changes

Reply:
Okay, I did this now